runtime/src/bank.rs

```diff
@@ -566,6 +577,8 @@ impl Bank {
         };
         inc_new_counter_debug!("bank-register_tick-registered", 1);
 
+        self.update_tick_height();
```

Comment:
@rob-solana - is this gonna mess with bank freezing?

Reply:
yes

Reply:
yep...  time to put that special-case code in...

Reply:
actually, looks like maybe not...  internal_state should be deterministic and store() does not update is_delta()

Reply:
#4500 has test updates that this PR would cause to fail ;)

Reply:
@rob-solana , excellent, I'll rebase and hopefully those failures will show me how to fix this up :)

Reply:
@rob-solana , do you mean the test would fail with this PR without the fix also in #4500 ? Tests are passing (see current CI run, rebased on master)
The tick_height syscall seems to be properly ignored in `hash_internal_state`, since it is owned by `syscall::id()`. Am I missing something else broken (untested)?

Reply:
everything should pass, now.  I ended up writing the fix at the same time.

Reply:
Good deal, thanks. R+? @mvines?